src/users.ts

```diff
@@ -0,0 +1,901 @@
+     * @param {String} token - Access token.
+     * @returns {Object} - Things object.
+     */
+    const stringParams: Record<string, string> = Object.fromEntries(
+      Object.entries(queryParams).map(([key, value]) => [key, String(value)])
+    )
+
+    const options: RequestInit = {
+      method: 'GET',
+      headers: {
+        'Content-Type': this.contentType,
+        Authorization: `Bearer ${token}`
+      }
+    }
+    try {
+      const response = await fetch(
+        new URL(
+          `${this.usersEndpoint}/${userId}/things?${new URLSearchParams(stringParams).toString()}`,
+          this.thingsUrl
+        ).toString(),
+        options
+      )
+      if (!response.ok) {
+        const errorRes = await response.json()
+        throw this.userError.HandleError(errorRes.error, response.status)
+      }
+      const thingsData: Things = await response.json()
+      return thingsData
+    } catch (error) {
+      throw error
+    }
+  }
+
+  public async ListUserChannels (
+    userId: string,
+    queryParams: QueryParams,
+    token: string
+  ): Promise<Channels> {
+    // Get channels of a user.
+    /**
+     * Gets the various channels a user owns.
+     * @method ListUserChannels - Get channels of a user.
+     * @param {String} userId - Member ID.
+     * @param {Object} queryParams - Query parameters for example offset and limit.
+     * @param {String} token - Access token.
+     * @returns {Object} - Channels object.
+     */
+
+    const stringParams: Record<string, string> = Object.fromEntries(
+      Object.entries(queryParams).map(([key, value]) => [key, String(value)])
+    )
+
+    const options: RequestInit = {
+      method: 'GET',
+      headers: {
+        'Content-Type': this.contentType,
+        Authorization: `Bearer ${token}`
+      }
+    }
+    try {
+      const response = await fetch(
+        new URL(
+          `${this.usersEndpoint}/${userId}/channels?${new URLSearchParams(stringParams).toString()}`,
+          this.thingsUrl
+        ).toString(),
+        options
+      )
+      if (!response.ok) {
+        const errorRes = await response.json()
+        throw this.userError.HandleError(errorRes.error, response.status)
+      }
+      const channelsData: Channels = await response.json()
+      return channelsData
+    } catch (error) {
+      throw error
+    }
+  }
+
+  public async ResetPasswordRequest (email: string): Promise<Status> {
+    // Sends a request to reset a password
+    /**
+     * @method ResetPasswordRequest - Sends a request
+     * @param {String} email - User email.
+     * @returns {Int} - Status.
+     * @example
+     * const user = {
+     * "credentials": {
+     *    "identity": "admin@example.com",
+     *   "password": "12345678"
+     * }
+     * }
+     */
+
+    const options: RequestInit = {
+      method: 'POST',
+      headers: {
+        'Content-Type': this.contentType
+      },
+      body: email
```

Comment:
this should be a json object
```suggestion
      body: JSON.stringify( { email: email } )
```